cmd/vulcan-trivy/entrypoint.sh

```diff
@@ -4,8 +4,9 @@
 
 set -e
 
-# untar trivy cache file
-tar xf trivy_cache.tgz
+if [ -d /root/.cache ]; then
```

Comment:
We could skip this checking the options env variable for `"force_update_db": true`